dusty/systems/docker/testing_image.py

```diff
@@ -5,6 +5,12 @@
 from ...compiler.compose import lib_install_commands_for_app_or_lib, container_code_path, get_volume_mounts
 from ...log import log_to_client
 
+def _remove_image_if_exists(docker_client, image_name):
+    try:
+        docker_client.remove_image(image=image_name)
+    except:
+        log_to_client('was not able to remove image {}'.format(image_name))
```

Comment:
yeah but exception handling isn't really a good candidate for throwing into a function imo. you're really just obfuscating what you're actually trying to do (`docker_client.remove_image` which is pretty straightforward) and adding some sneaky exception handling that isn't really hinted at by the name of the function

The "meat" of the function is only one line, and you're actually making it more complex and harder to reason about by abstracting it, so it seems like a bad fit